lib/config/configuration.go

```diff
@@ -346,8 +346,10 @@ func ApplyFileConfig(fc *FileConfig, cfg *service.Config) error {
 
 	// Read in how nodes will validate the CA. A single empty string in the file
 	// conf should indicate no pins.
-	if len(fc.CAPin) > 1 || (len(fc.CAPin) == 1 && fc.CAPin[0] != "") {
-		cfg.CAPins = fc.CAPin
+	if len(fc.CAPin) > 0 {
```

Comment:
I just wanted to avoid the make() in ApplyCAPins, we can push the check into the function though. And below I forgot to check the error - adding that too.